internal/pkg/cgroups/util.go

```diff
@@ -117,12 +118,25 @@ func HasXDGRuntimeDir() (bool, error) {
 }
 
 // CanUseCgroups checks whether it's possible to use the cgroups manager.
+// - Systemd cgroups management requires systemd running as init.
 // - Host root can always use cgroups.
 // - Rootless needs cgroups v2.
 // - Rootless needs systemd manager.
 // - Rootless needs DBUS_SESSION_BUS_ADDRESS and XDG_RUNTIME_DIR set properly.
 // warn controls whether configuration problems preventing use of cgroups will be logged as warnings, or debug messages.
 func CanUseCgroups(systemd bool, warn bool) bool {
+	if systemd {
+		systemdRunning := fs.IsDir("/run/systemd/system")
+		if !systemdRunning {
+			if warn {
+				sylog.Warningf("Cannot use systemd cgroups manager, systemd not running as init on this host.")
+			} else {
+				sylog.Warningf("Cannot use systemd cgroups manager, systemd not running as init on this host.")
+			}
```

Comment:
Both cases have exactly the same warning message. That doesn't seem right. 

Or Am I missing something?

Reply:
Based on the comment one should be a debug log

Reply:
:facepalm: meant to be same message, but bottom one Debug.